src/borg/archiver/prune_cmd.py

```diff
@@ -18,16 +26,15 @@
 logger = create_logger()
 
 
-def prune_within(archives, seconds, kept_because):
-    target = datetime.now(timezone.utc) - timedelta(seconds=seconds)
-    kept_counter = 0
-    result = []
-    for a in archives:
-        if a.ts > target:
-            kept_counter += 1
-            kept_because[a.id] = ("within", kept_counter)
-            result.append(a)
-    return result
+def unique_func():
```

Comment:
I think this deserves some comments to explain what is going on, both here and when used.

Reply:
I added a comment on how the period grouping functions work in general and changed the wording of the default function to match the others in descriptiveness. Let me know what you think.